templates/module/Source/module/Private/Generated/OLink/olinkclient.cpp.tpl

```diff
@@ -31,14 +31,34 @@ THIRD_PARTY_INCLUDES_START
 THIRD_PARTY_INCLUDES_END
 
 {{- if len .Interface.Properties }}
+    {{- $shouldIncludeAtomic := 0 -}}
```

Comment:
The indention looks off. Just move it to the beginning of the line.

side note:
Also I am wondering whether we can move that to some kind of "macro".
We have similar complex issues in the template and it always looks kinda ugly